tests/ttnn/unit_tests/operations/test_conv2d.py

```diff
@@ -988,6 +996,7 @@ def test_unet_conv(
     )
 
 
+@skip_for_wormhole_b0("Issue #6989: AttributeError: 'NoneType' object has no attribute 'enable_program_cache'")
```

Comment:
Unet convs in UNet model runs on WH without errors. Not sure why unit test is failing. @nsmithtt we don't see an error on WH right?